core/src/main/scripts/migrate_db.py

```diff
@@ -150,6 +154,32 @@ def is_version_larger(version1, version2):
         return True
     return False
 
+def check_reference_genome(portal_properties, cursor, force_migration):
+    """ Retrieve reference genomes from database """
+    try:
+        sql_statement = """
+                           select count(distinct NCBI_BUILD), CANCER_STUDY_IDENTIFIER
+                           from mutation_event 
+                           join mutation on mutation.MUTATION_EVENT_ID = mutation_event.MUTATION_EVENT_ID 
+                           join genetic_profile on genetic_profile.GENETIC_PROFILE_ID = mutation.GENETIC_PROFILE_ID
+                           join cancer_study on cancer_study.CANCER_STUDY_ID = genetic_profile.CANCER_STUDY_ID
+                           where NCBI_BUILD not in ('NA','')
```

Comment:
The sql statement in the migration.sql script does not filter out values 'NA' or '' ... so I think the test for problems here should also not filter out these strings. Or maybe the query inside the migration script should also filter out these values when trying to choose a build.

Reply:
agreed ^

Reply:
all right, will not filter out 'NA' or ''.

Reply:
done.